### PR TITLE
Fix up static links for root pages

### PR DESCRIPTION
**Pull request by cooperaj**
## Purpose
Fixes up links across the site to the static privacy policy, cookies and terms of use pages. Sets them up to properly interpret the current language on the site and set links accordingly.

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [x] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

**Review by williamfalconeruk — APPROVED** [review]

**Review by nickdavis2001 — APPROVED** [review]
lgtm

File: service-front/app/src/Viewer/templates/viewer/enter-code.html.twig [hunk 1]
```diff
@@ -51,7 +51,7 @@
                             {% trans %}An <abbr title="lasting power of attorney">LPA</abbr> access code is a unique code given to you by the donor or an attorney named on a lasting power of attorney.{% endtrans %}
                         </div>
 
-                        <p class="govuk-body">{% trans with {'%link%': path('viewer-terms-of-use')} %}By continuing you agree to our <a href="%link%" class="govuk-link">terms of use</a>.{% endtrans %}</p>
+                        <p class="govuk-body">{% trans with {'{link}': path('terms-of-use')} %}By continuing you agree to our <a href="{link}" class="govuk-link">terms of use</a>.{% endtrans %}</p>
```
Comment (williamfalconeruk) [inline service-front/app/src/Viewer/templates/viewer/enter-code.html.twig:54]: is this the right format for `trans with link`?

Comment (cooperaj) [reply service-front/app/src/Viewer/templates/viewer/enter-code.html.twig:54]: it works but is not the way we've been doing it. good spot

Comment (williamfalconeruk) [reply service-front/app/src/Viewer/templates/viewer/enter-code.html.twig:54]: i think just needs a minor tweak back to using the `%` as we use this elsewhere.

Comment (cooperaj) [reply service-front/app/src/Viewer/templates/viewer/enter-code.html.twig:54]:
```suggestion
                        <p class="govuk-body">{% trans with {'%link%': path('terms-of-use')} %}By continuing you agree to our <a href="%link%" class="govuk-link">terms of use</a>.{% endtrans %}</p>
```

**Review by williamfalconeruk — CHANGES_REQUESTED** [review]
one minor tweak, then GTG.

**Review by williamfalconeruk — APPROVED** [review]

**Review by nickdavis2001 — APPROVED** [review]

**Review by SeemaMenon — APPROVED** [review]

**Pull request merged**